### PR TITLE
Force MenuDropAlignment on startup

### LINKED ISSUE #1374: SCMU menu dropdown menus are misaligned
After upgrading to 3.0.0, I'm seeing the following in SCMU:

image1
image2
image3

@danielmarbach has said he is seeing the same behavior, but @boblangley is not seeing it on his computer.

### PR DESCRIPTION
**Pull request by tmasternak**
fixes #1374

### CODE REVIEW

**Comment by tmasternak** [comment]
@HEskandari could you have a look an see if this makes sense? We need your WPF skills 🦇

**Comment by HEskandari** [comment]
@tmasternak I think it is also good to ask @mikeminutillo to see if it really fixes the issue, as he can consistently reproduce that issue on his machine, which can't be said for me. 

I'm more than happy to review the code though.

**Comment by janovesk** [comment]
@tmasternak could both repo and see that the fix worked on his tablet/latop @HEskandari

File: src/ServiceControl.Config/App.xaml.cs [hunk 1]
```diff
@@ -16,6 +17,23 @@ public App()
             {
                 ExceptionMessageBox.Attach();
             }
+
+            EnsureStandardPopupAlignment();
+        }
+
+        static void EnsureStandardPopupAlignment()
```
Comment (HEskandari) [inline src/ServiceControl.Config/App.xaml.cs]: A bit nitpicking here, but could you put the logic in a class and use it in App? This is a workaround and clutters the App.cs code. e.g.

```StandardPopup.ApplyAlignment();```

The  hint/comment can also be moved into the class implementing the workaround.

Comment (tmasternak) [reply src/ServiceControl.Config/App.xaml.cs]: @HEskandari moved.

**Pull request merged**